progressbar.go

```diff
@@ -31,30 +30,30 @@ func NewProgressBar(prefix string) desync.ProgressBar {
 		bar.Callback = func(s string) { fmt.Fprintln(os.Stderr, s) }
 		bar.Output = nil
 	}
-	return ProgressBar{bar}
+	return ProgressBarDesync{bar}
 }
 
-// ProgressBar wraps https://github.com/cheggaaa/pb and implements desync.ProgressBar
-type ProgressBar struct {
+// ProgressBarDesync wraps https://github.com/cheggaaa/pb and implements desync.ProgressBar
```

Comment:
Wrt naming, would `DefaultProgressBar` work as well? Not ideal to suffix it with `Desync` tbh.